db/re/enchantgrade.yml

```
@@ -0,0 +1,213 @@
+# This file is a part of rAthena++.
+#   Copyright(C) 2022 rAthena Development Team
+#   https://rathena.org - https://github.com/rathena
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+###########################################################################
+# Enchantgrade Database
+###########################################################################
+#
+# Enchantgrade Settings
+#
+###########################################################################
+# - Type                                Item type.
+#   Levels:                             Enchantgrade settings per item level.
+#     - Level                           Item level.
+#       Grades:                         Enchantgrade settings per refine level.
+#         - Grade                       Enchantgrade level.
+#           Refine                      Required refine level
+#           Chance                      Base chance of success
+#           Bonus                       Enchantgrade bonus. (Default: 0)
+#           Announce                    Announce if someone tries to increase the enchantgrade? (Default: false)
+#           Catalysator:                Catalysator item to increase your chance of success
+#             Item                      The item that can be used
+#             AmountPerStep             How many of the specified items are needed
+#             MaximumSteps              How many many items you can use maximum
+#             ChanceIncrease            How much the chance increases for each item used
+#           Options:                    Success chance based on cost type.
+#             - Option                  Index of the client option
+#               Item                    Required item.
+#               Price                   Amount of zeny required. (Default: 0)
+#               BreakingRate            Chance of item breaking out of 0~10000. (Default: 0)
+#               DowngradeAmount         Number of refine levels reduced on failure. (Default: 0)
+###########################################################################
+
+Header:
+  Type: ENCHANTGRADE_DB
+  Version: 1
+
+Body:
+  - Type: Armor
+    Levels:
+      - Level: 2
+        Grades:
+          - Grade: 0
```

How about adding a constant for each grade, since each grade level is designed by D C B A in the item description., so we know what is what.
Suggestion : constant GRADE_D for grade level "0"
```suggestion
          - Grade: D
```

---

Grade D = 1, but I added your suggestion.